### PR TITLE
fix(tooltip): Allow aria-label pass-through in muted tooltips

### LINKED ISSUE #1036: IconButtons wrapped by muted tooltips do not respect aria-label set on button
## 🐛 Bug Report

With a muted tooltip wrapping the IconButton, the aria-label is undefined and no aria-label appears on the button element.

## To Reproduce

Without tooltip:
```
<IconButton
            aria-label="Activity Stream"
            icon={activityStreamIcon}
            size={IconButton.Size.Medium}
            variant={IconButton.Variant.InverseFilled}
          />
```
renders
```
<button aria-label="Activity Stream" > ... </button>
```

With tooltip:
```
    <Tooltip title={"some title"} type="muted">
        <IconButton
          aria-label="Activity Stream"
          variant={IconButton.Variant.Plain}
          size={IconButton.Size.Medium}
          icon={icon}
          onClick={onClick}
        />
    </Tooltip>
```
renders
```
<button> ... </button>
```

## Expected Behavior

An IconButton should have an aria-label attribute set when wrapped in a muted tooltip.

## Actual Results

An IconButton wrapped in a muted tooltip does not have an aria-label attribute.

## Browser (if applicable)

If this was a visual bug, what browser did you spot it on?

## Link to repl or repo (highly encouraged)

Please provide a reference to a repo or branch to help us reproduce this bug.

### Error Output

Paste the error output here

```bash

```

or

Include a screenshot of a visual bug

### PR DESCRIPTION
**Pull request by NicholasBoll**
Fixes #1036

### CODE REVIEW

**Review by csongnguyen — APPROVED** [review]
Thank you Nicholas! 🎉

File: modules/tooltip/react/lib/Tooltip.tsx [hunk 1]
```diff
@@ -76,6 +76,9 @@ export const Tooltip = ({
       {React.cloneElement(children, {
         ...targetProps,
         ...mergeCallbacks(children.props, targetProps),
+        ...(type === 'muted' && children.props['aria-label']
+          ? {'aria-label': children.props['aria-label']}
+          : {}),
```
Comment (alanbsmith) [inline modules/tooltip/react/lib/Tooltip.tsx:81]: 👍🏻

**Review by alanbsmith — APPROVED** [review]
Looks good. Thanks for the fix + test. 🥇

File: modules/tooltip/react/lib/Tooltip.tsx (hunk 1 above)
Comment (NicholasBoll) [reply modules/tooltip/react/lib/Tooltip.tsx:81]: Definitely messy, but this should hold over v4. Things will change in future versions. I'm not sure there will be a "muted" type, but a muted pattern.

**Pull request merged**